canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -167,6 +172,25 @@ class RadiosetControl extends React.Component {
 		let wasChecked = false;
 		const valueSet = this.props.controlOpts;
 		for (var i = 0; i < valueSet.values.length; i++) {
+			if (this.props.control.valueDescs && !(disabledProps) && !this.props.tableControl) {
```

Comment:
This should be moved like checkboxset and the disabled logic should be the same as the radio button.  Right now the tooltip can be disabled but the radio but might be disabled.

Reply:
Updated the code like checkbox.